psd-web/spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+require "support/matchers/capybara_matchers"
+
+RSpec.configure do |config|
+  config.include PageMatchers
+end
+
+def have_summary_error(text)
+  have_css(".govuk-error-summary__list", text: text)
+end
+
+def expect_confirmation_banner(msg)
+  expect(page).to have_css(".hmcts-banner__message", text: msg)
+end
+
+def expect_h1_on_the_page(header)
```

Comment:
```suggestion
def expect_page_to_have_h1(header)
```
Matches the standard syntax better.